site/content/docs/user/quick-start.md

```diff
@@ -150,12 +150,13 @@ context name `kind` and delete that cluster.
 
 ## Loading an Image Into Your Cluster
 
-Docker images can be loaded into your cluster nodes with:
-`kind load docker-image my-custom-image`
+Docker images can be loaded into your cluster nodes by specifying their names as a space-separated list:
```

Comment:
the "space-seperate list" bit should also be dropped here
```suggestion
Docker images can be loaded into your cluster nodes with:
```
the example and `--help` text should be sufficient to define the format (and again, it may not be spaces, it's actually controlled by `$IFS` in most shells, and in a calling program it will be an argument array)

Reply:
Aah, forgot to update this one :see_no_evil